### PR TITLE
3530: Provide a help text on the empty conversations view

### LINKED ISSUE #3530: Show help text on empty private messages
Followup of #3408 

The private message view is one of the three initial views which are empty (for a new account).

Private messages are non-obvious for beginners (e. g. how to create one) so there should be a short help text in that case.

### PR DESCRIPTION
**Pull request by Lakoja**
Fixes #3530

### CODE REVIEW

File: app/src/main/java/com/keylesspalace/tusky/components/conversation/ConversationsFragment.kt [hunk 1]
```diff
@@ -135,6 +135,7 @@ class ConversationsFragment :
                         if (loadState.append is LoadState.NotLoading && loadState.source.refresh is LoadState.NotLoading) {
                             binding.statusView.show()
                             binding.statusView.setup(R.drawable.elephant_friend_empty, R.string.message_empty, null)
+                            binding.statusView.showHelp(R.string.help_empty_conversations)
```
Comment (Lakoja) [inline app/src/main/java/com/keylesspalace/tusky/components/conversation/ConversationsFragment.kt:138]: This is nicely simple now.

**Comment by Lakoja** [comment]
> Error: Replace straight quotes (") with directional quotes (“”, “ and ”) ?

That is an error?

**Comment by Lakoja** [comment]
Note: source branch changed (further two commits) which is not shown here...

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -812,4 +812,15 @@
...
+    <string name="help_empty_conversations">Here are your <b>private messages</b>; sometimes called conversations or direct messages (DM).
+        \n\nPrivate messages are created by setting the visibility [iconics gmd_public] of a post to [iconics gmd_mail] \"Direct\" and
+        mentioning one or more users in the text.
+        \n\nFor example you can start on the profile view of an account and tap the create button [iconics gmd_edit] and change the visibility.
+    </string>
+    <string name="help_empty_lists">This is your <b>lists view</b>. You can define a number of private lists and add accounts to that.
+        \n\n
+        NOTE that you can only add accounts you follow to your lists.
+        \n\n
+        These lists can be used as a tab in Account preferences [iconics gmd_account_circle] [iconics gmd_navigate_next] Tabs.
```
Comment (Lakoja) [inline app/src/main/res/values/strings.xml:824]: gmd_account_circle is not the correct one here.

It should be manage_account. But that is not present in GoogleMaterial.kt.

Comment (Lakoja) [reply app/src/main/res/values/strings.xml:824]: The respective file is 3 years old: https://github.com/mikepenz/Android-Iconics/blob/develop/google-material-typeface/google-material-typeface-library/src/main/java/com/mikepenz/iconics/typeface/library/googlematerial/GoogleMaterial.kt

**Review by Tak — APPROVED** [review]

**Pull request merged**